pytest_blockage.py

```diff
@@ -39,6 +44,34 @@ def whitelisted(self, host, *args, **kwargs):
         httplib.HTTPConnection.__init__ = whitelisted
 
 
+def block_http_aiohttp(whitelist):
+    try:
+        from aiohttp.client import ClientSession
+    except ImportError:
+        # no aiohttp installed
+        return
+
+    from yarl import URL
+
+    def patch(self, method, url, **kwargs):
```

Comment:
I am fine with naming the patched method patch, but if you do it, please rename whitelisted above too. It should be consistent.